frappe/core/doctype/communication/communication.py

```diff
@@ -48,25 +48,25 @@ def validate(self):
 	def after_insert(self):
 		if not (self.reference_doctype and self.reference_name):
 			return
-
-		if self.communication_type in ("Communication", "Comment"):
-			# send new comment to listening clients
-			frappe.publish_realtime('new_communication', self.as_dict(),
-				doctype= self.reference_doctype, docname = self.reference_name,
-				after_commit=True)
-
-			if self.communication_type == "Comment":
-				notify_mentions(self)
-
-		elif self.communication_type in ("Chat", "Notification", "Bot"):
-			if self.reference_name == frappe.session.user:
-				message = self.as_dict()
-				message['broadcast'] = True
-				frappe.publish_realtime('new_message', message, after_commit=True)
-			else:
-				# reference_name contains the user who is addressed in the messages' page comment
-				frappe.publish_realtime('new_message', self.as_dict(),
-					user=self.reference_name, after_commit=True)
+		if not self.timeline_hide:
```

Comment:
timeline hide is the field for hiding duplicate emails ie user sends an email to 2 other users in the system rather than display 3 emails in the timeline timeline_hide displays one